tests/lib/rules/no-multiple-empty-lines.js

```diff
@@ -82,7 +82,7 @@ ruleTester.run("no-multiple-empty-lines", rule, {
             options: ruleArgs
         },
         {
-            code: "// invalid 3\nvar a=5;\n\n\n\nvar b = 3;",
+            code: "// invalid 3\nvar a=5;\n\n\n\nvar b = 3;\n",
```

Comment:
Just for clarification, with the added empty line, the code was not detected as invalid anymore.